backend/sirius-web-spring-collaborative-diagrams/src/main/java/org/eclipse/sirius/web/spring/collaborative/diagrams/dto/ExportRepresentationPayload.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.spring.collaborative.diagrams.dto;
+
+import java.text.MessageFormat;
+import java.util.Objects;
+import java.util.UUID;
+
+import org.eclipse.sirius.web.core.api.IPayload;
+
+/**
+ * The payload of the export representation handler.
+ *
+ * @author rpage
+ */
+public final class ExportRepresentationPayload implements IPayload {
+    private final UUID id;
+
+    private final String name;
+
+    private final String svgExport;
```

Comment:
Use `content` instead of `SVGExport`, the API provided to the frontend is related to the representation export, it could be used to export representations to another format